tests/functional/reset_password.js

```diff
@@ -128,18 +128,38 @@ define([
     },
 
     'open /reset_password page from /signin': function () {
+      const updatedEmail = 'testuser@testuser.com';
```

Comment:
nit: `testuser.com` a bit worried about that if this test evolves we might start sending traffic to that domain/email